web_src/less/helpers.less

```diff
@@ -18,6 +18,7 @@
 .w-100 { width: 100% !important; }
 .h-100 { height: 100% !important; }
 .br-0 { border-radius: 0 !important; }
+.pen { pointer-events: none !important; }
```

Comment:
Shouldn't this rather be called `no-pen` or `no-pointer`?

Reply:
No, according to current scheme it's fine:

`pen` is `pointer-events: none`
`pea` would be `pointer-events: auto`